src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.cards {
+  display: grid;
+  grid-template-columns: 200px;
+  gap: 46px 48px;
+  justify-content: center;
+
+  margin-bottom: 72px;
+  padding: 50px 40px 0;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  width: 200px;
+  padding: 32px 16px 0;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  transition: all 300ms ease-in;
```

Comment:
Avoid using 'all' in the transition property. Specify which properties you want to apply the transition to, such as 'transform' or 'color'.